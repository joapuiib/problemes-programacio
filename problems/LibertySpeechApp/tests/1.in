3
Lorem 12 2.5
Ipsum 55 6
Dolor 3 10

