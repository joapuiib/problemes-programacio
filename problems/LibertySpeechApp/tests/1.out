---------- Paisos ----------
Lorem - casos: 12 - gravetat: 2.5 - puntuació: 30.0
Ipsum - casos: 55 - gravetat: 6.0 - puntuació: 330.0
Dolor - casos: 3 - gravetat: 10.0 - puntuació: 30.0
---------- Resum ----------
Casos totals: 70
País amb més casos: Ipsum
